website/integrations/services/bookstack/index.md

```diff
@@ -18,53 +18,56 @@ This is based on authentik 2021.7.2 and BookStack V21.05.3. Instructions may dif
 
 The following placeholders will be used:
 
--   `book.company` is the FQDN of BookStack.
+-   `bookstack.company` is the FQDN of BookStack.
 -   `authentik.company` is the FQDN of authentik.
--   `METADATAURL` is the url for the SAML metadata from authentik
+-   `METADATAURL` is the URL for the SAML metadata from authentik.
+
+## authentik Configuration
 
 ### Step 1
 
-In authentik, under _Providers_, create a _SAML Provider_ with these settings:
+In authentik, under **Providers**, create a **SAML Provider** with these settings:
 
 **Protocol Settings**
 
--   Name: Bookstack
--   ACS URL: https://book.company/saml2/acs
--   Issuer: https://authentik.company
--   Service Provider Binding: Post
--   Audience: https://book.company/saml2/metadata
+-   **Name:** Bookstack
+-   **ACS URL:** `https://bookstack.company/saml2/acs`
+-   **Issuer:** `https://authentik.company`
+-   **Service Provider Binding:** Post
+-   **Audience:** `https://bookstack.company/saml2/metadata`
 
-**Advanced protocol settings**
+**Advanced Protocol Settings**
 
--   Signing Certificate: Choose your certificate or the default authentik Self-signed Certificate
-    All other options as default.
+-   **Signing Certificate:** Choose your certificate or the default authentik Self-signed Certificate. All other options should remain as default.
 
-![](./authentik_saml_bookstack.png)
+![authentik_saml_bookstack](./authentik_saml_bookstack.png)
 
 ### Step 2
 
-In authentik, create an application which uses this provider. Optionally apply access restrictions to the application using policy bindings.
+In authentik, create an application that uses this provider. Optionally apply access restrictions to the application using policy bindings.
 
--   Name: Bookstack
--   Slug: bookstack
--   Provider: Bookstack
--   Launch URL: https://book.company
+-   **Name:** Bookstack
+-   **Slug:** bookstack
+-   **Provider:** Bookstack
+-   **Launch URL:** `https://bookstack.company`
 
 ### Step 3
 
 Obtain your Metadata URL from authentik.
 
-1. Click on the BookStack Provider
-2. Click the Metadata Tab
-3. Click Copy download URL (This URL is the `METADATAURL` required in Step 4)
+1. Click on the BookStack Provider.
```

Comment:
Ideally we don't say "on the" we just say "Click **BookStack Provider**." but... if further clarification needed ou could say something like ""Click **BookStack Provider** in the upper right."